Hunting Queries/AzureActivity/AzureNSG_AdministrativeOperations

```diff
@@ -0,0 +1,30 @@
+id: e94d6756-981c-4f02-9a81-d006d80c8b41
+name: Azure Network Security Group NSG Administrative Operations
+description: |
+  'Identifies set of Azure NSG administrative operational detection queries for hunting activites'
+severity: Low
+requiredDataConnectors:
+  - connectorId: AzureActivity
+    dataTypes:
+      - AzureActivity
+queryFrequency: 1d
+queryPeriod: 7d
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Impact
+relevantTechniques:
+  - T1496
+query: | 
+ 
+let timeframe = 1d; 
+// Azure NSG Create / Update / Delete
+AzureActivity
+| where Category == "Administrative"
+| where TimeGenerated >= ago(timeframe)
+| where OperationNameValue in ("Microsoft.Network/networkSecurityGroups/write", "Microsoft.Network/networkSecurityGroups/delete")
```

Comment:
As this is a list, you may want to consider parameterizing this and placing this above the query body as you ahve done with timeframe - let opValues = dynamic(["Microsoft.Network/networkSecurityGroups/write", "Microsoft.Network/networkSecurityGroups/delete"]);